vlang/build.sh

```diff
@@ -0,0 +1,13 @@
+TERMUX_PKG_HOMEPAGE=https://vlang.io/
+TERMUX_PKG_DESCRIPTION="V programming language compiler"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=0.4.12
+TERMUX_PKG_SRCURL=https://github.com/vlang/v/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=65e3579df61ae0a7314aa5f0c7eb3b0b9b45170a0275e392e9c168be23046e89
+TERMUX_PKG_DEPENDS="build-essential, clang, make, libgc-dev, libexecinfo-dev, libatomic1"
```

Comment:
These depends don't make any sense and don't match dependencies for any other package of `vlang`.
https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=vlang-bin#n16
https://build.opensuse.org/projects/openSUSE:Factory/packages/vlang/files/vlang.spec
https://github.com/Homebrew/homebrew-core/blob/6c4e6736161b14b1345bc48990f8e56d4039fe42/Formula/v/vlang.rb#L23-L27

- `libgc` seems to be correct, but we do not provide separate `*-dev` packages for headers, that's a Debian-ism.
- `build-essential`, `clang` and `make` are provided as base components of the build environment.
- I have no idea where you got `libexecinfo` or `libatomic1` as dependencies from.